.vscode/settings.json

```diff
@@ -0,0 +1,22 @@
+{
+    "esbonio.sphinx.confDir": "${workspaceFolder}/copter/source",
+    "grammarly.config.documentDialect": "american",
+    "editor.wordWrap": "on",
+    "grammarly.files.include": [
+        "**/readme.md",
+        "**/README.md",
+        "**/*.txt",
+        "**/*.rst"
+    ],
+    "files.exclude": {
+        "antennatracker/**/common-*": true,
+        "blimp/**/common-*": true,
+        "copter/**/common-*": true,
+        "dev/**/common-*": true,
+        "mavproxy/**/common-*": true,
+        "plane/**/common-*": true,
+        "planner/**/common-*": true,
+        "planner2/**/common-*": true,
+        "rover/**/common-*": true,
+    }
+}
```

Comment:
```suggestion
}

```

Reply:
Whoops! For some reason VSCode settings json formatter doesn't add those in automatically. I'll fix it in a few hours when I get to a desktop.